test/draw.py

```diff
@@ -56,49 +33,68 @@ class TestDraw(unittest.TestCase):
 
     # Result image for draw
     def test_draw(self):
-        image = Image.new(imageMode, imageSize, imageBackground)
-        canvas = ImageDraw.Draw(image)
 
-        for i in range(len(points)):
-            draw(canvas, (points[0], points[1]), 'A')
+        """ Create the file name and its saving path, and specify the reference file to compare to."""
+        image_name = filename(sys._getframe().f_code.co_name)
+        result_file, reference_file = get_path(image_name)
+
+        ''' This function is to create an empty image with a specific dimension
```

Comment:
what's with this?

we should only have one help message for each function/module.